CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## 10.3.2
  - [DOC] Replaced setting name with correct value [#919](https://github.com/logstash-plugins/logstash-output-elasticsearch/pull/919)
+ - Fix integration tests for Elasticsearch 7.6+ [#922](https://github.com/logstash-plugins/logstash-output-elasticsearch/pull/922)
```

Comment:
Can we also include the second PR which fixes integration tests for `7.5.0` of the Elasticsearch API
```suggestion
 - Fix integration tests for Elasticsearch 7.6+ [#922](https://github.com/logstash-plugins/logstash-output-elasticsearch/pull/922)
 - Fix integration tests for Elasticsearch API `7.5.0` [#923](https://github.com/logstash-plugins/logstash-output-elasticsearch/pull/923)
```